### PR TITLE
Diagram edits

### PR DESCRIPTION
**Pull request by kathryn-ods**
# Overview

- What does this pull request do? minor changes to docs - bringing diags in line with style guide and correcting a few things
- How can a reviewer test or examine your changes? build the branch and check trusts and SOE pages
- Who is best placed to review it? @Blueskies00 @kd-ods 

(Closes/Relates to) issue: #456 

## Translations

- [ ] New or edited strings appearing as a result of this PR will be picked up for translation
- [ ] I've notified the translation coordinator of any new strings that will need
      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

- [x] I've checked that the documentation builds without failing. See: https://github.com/openownership/data-standard#building-the-documentation-locally
- [ ] I've thought about how and when this needs to be released. See:
      https://openownership.github.io/bods-dev-handbook/standard_releases.html      
- [ ] I've updated the changelog, if necessary.
- [ ] I've updated [reference.rst](https://standard.openownership.org/en/latest/schema/reference.html) to reflect any changes to schema structure or naming.
- [ ] I've added or edited any other related documentation. See: https://github.com/openownership/bods-dev-handbook/blob/master/style_guide.md

### CODE REVIEW

**Comment by kathryn-ods** [comment]
@kd-ods I haven't figured out exactly why there was a conflict but have fixed it now

**Review by kd-ods — CHANGES_REQUESTED** [review]
@kathryn-ods - these images are looking much better!

Two things:

- You don't need to work on the translated SVGs by hand. Really sorry if I misled you about this. They get generated as part of the translation process later down the line. Ping me if you want to chat about how that will work.
- There's one diagram that's been missed: RepresentingChainsBODS-Statements.svg at the bottom of [this page](https://standard.openownership.org/en/main/schema/guidance/repr-beneficial-ownership.html)

**Comment by kathryn-ods** [comment]
> You don't need to work on the translated SVGs by hand. Really sorry if I misled you about this. They get generated as part of the translation process later down the line. Ping me if you want to chat about how that will work.
Not to worry! You didn't mislead me - For some reason I thought maybe diagrams that had already been translated wouldn't be translated again. Would definitely be good to talk through the process more soon. 

> There's one diagram that's been missed: RepresentingChainsBODS-Statements.svg at the bottom of [this page](https://standard.openownership.org/en/main/schema/guidance/repr-beneficial-ownership.html)
Thanks for spotting this one! I have updated it now :)

File: docs/_assets/RepresentingChainsBODS-Statements.svg
Comment (kd-ods) [inline docs/_assets/RepresentingChainsBODS-Statements.svg:1]: The 'Entity statement' type text above the statement boxes is misaligned.

**Review by kd-ods — CHANGES_REQUESTED** [review]
See comment: text is misaligned on the newly updated diagram.

File: docs/_assets/RepresentingChainsBODS-Statements.svg
Comment (kathryn-ods) [reply docs/_assets/RepresentingChainsBODS-Statements.svg:1]: Inkscape acting up again - fixed now!

**Review by kd-ods — APPROVED** [review]
Great! Ready to merge.

**Pull request merged**